Duplicati/Library/Backend/S3/S3Wrapper.cs

```diff
@@ -80,8 +79,10 @@ public S3Wrapper(string awsID, string awsKey, string locationConstraint, string
 
         public void AddBucket(string bucketName)
         {
-            PutBucketRequest request = new PutBucketRequest();
-            request.BucketName = bucketName;
+            PutBucketRequest request = new PutBucketRequest

```

Comment:
This and the others could be changed to `var` ?

Reply:
yes.